app/decorators/case/sar/offender_decorator.rb

```diff
@@ -5,7 +5,7 @@ def sar_response_address
   end
 
   def subject_type_display
-    object.subject_type.humanize.tr(' ', '-')
+    I18n.t('helpers.label.offender_sar.subject_type.'+object.subject_type)
```

Comment:
Tiny thing, sorry, please could we have spaces around the operator here?

```suggestion
    I18n.t('helpers.label.offender_sar.subject_type.' + object.subject_type)
```

Reply:
Yep, no problem